src/Microsoft.Performance.SDK.Runtime.Tests/CustomDataSourceReferenceExtensionsTests.cs

```diff
@@ -0,0 +1,281 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Microsoft.Performance.SDK.Processing;
+using Microsoft.Performance.Testing;
+using Microsoft.Performance.Testing.SDK;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using DataSourceAttribute = Microsoft.Performance.SDK.Processing.DataSourceAttribute;
```

Comment:
Why do we need this line if we have line 7?

Reply:
Yes, the test framework also has a DataSourceAttribute.